### DISCUSSION TITLE
Enhance Tag Functionality

### DISCUSSION THREAD

**Discussion by rebelmx4** [spacedriveapp/spacedrive#2582] [Feature Request]
Hi,

I love using your project, but I would find it more useful if it supported tag groups for organizing multiple tags. For example:

- **Family**:
  - Me, my wife, my child, my father, my mother
- **Location**:
  - Park, market, rural
- ...

This feature would greatly enhance usability and organization.

Thank you for considering this request.

Best regards

**Comment by fvnju** [comment]
I agree. I also feel like there should be tags specific to a folder like in Eagle file manager [](https://en.eagle.cool/)

**Comment by augviolet** [comment]
yes！There is a great need for a more robust labeling system, including grouping of labels, multi-layer labeling, and batch labeling.
